File: exercicios/cpp/Arrays/Q10.cpp
/*
------------------------------------------------
10. Crie um algoritmo/fluxograma/programa que 
substitua todos os elementos negativos em um 
array por zero.
 */
#include <iostream>
#include <clocale>
#include <cstdlib>

using namespace std;
int main() {
	setlocale(LC_ALL,"Portuguese");
	int dado[] = {-3, 0, 8, -1, 13, 7, 34, -6};
	const int tam = sizeof dado/sizeof dado[0];

	for (int i = 0; i < tam; i++)
		if (dado[i] < 0)
			dado[i] = 0;

	for (int i = 0; i < tam; i++)
		cout<<dado[i]<<" ";
	cout<<endl;
			
	return EXIT_SUCCESS;	
}

File: exercicios/python/Q10.py
# Exercício 10: Substitua elementos negativos por zero
lista = [5, -3, 7, -1, 0, -5, 2]
print("Antes:", lista)
i = 0
while i < len(lista):
    if lista[i] < 0:
        lista[i] = 0
    i += 1
print("Depois:", lista)